lib/elixir_jobs/offers/offer.ex

```diff
@@ -38,7 +39,8 @@ defmodule ElixirJobs.Offers.Offer do
     |> validate_required(@required_attrs)
     |> validate_length(:title, min: 5, max: 50)
     |> validate_length(:company, min: 2, max: 30)
-    |> validate_length(:description, min: 10, max: 500)
+    |> validate_length(:description, min: 10, max: 1000)
+    |> validate_length(:summary, max: 200)
```

Comment:
... and I would also put a minimum to avoid descriptions with, for example, just a letter/point. What do you think about a minimum of 10 characters?